index.html

```diff
@@ -13,14 +13,48 @@
 
 	</head>
 	<body>
```

Comment:
@dlevkovits Good job with the indentation on this file! Easy to navigate and understand :)